docs/changelog/index.html

```diff
@@ -0,0 +1,164 @@
+          </div>
+
+          <button class="navbar-toggle collapsed" type="button" data-toggle="collapse" data-target="#primary-navigation"
+            aria-controls="primary-navigation" aria-expanded="false">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+        </div>
+        <ul class="nav navbar-nav navbar-right">
+          <li>
+            <a class="page-link" href="/changelog/">Changelog</a>
+          </li>
+          <li>
+            <a class="page-link" href="/samples/">Samples</a>
+          </li>
+          <li>
+            <a class="page-link" href="https://developer.webex.com/docs/sdks/browser#samples">Developer Portal</a>
+          </li>
+        </ul>
+      </div>
+    </header>
+    <main>
+      <div class="container">
+        <div class="search-container">
+            <form id="search-form" method="GET">
+                <div class="form-row">
+                    <div class="form-group">
+                        <label for="version-select">Select Stable Version:</label>
+                        <select id="version-select" class="full-width">
+                            <option value="">Select a version</option>
+                            <!-- Options will be populated here -->
+                        </select>
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-group" id="package-input-group">
+                        <label for="package-name-input">Package Name:</label>
+                        <!-- <input type="text" id="package-name-input" class="full-width"> -->
+                        <select id="package-name-input" class="full-width">
+                            <!-- Options will be populated here -->
+                        </select>
+                    </div>
+                    <div class="form-group" id="version-input-group">
+                        <label for="version-input">Version:</label>
+                        <input type="text" id="version-input" class="full-width">
+                        <span style="color:red" id="version-input-error"></span>
+                    </div>
+                    <div class="form-group" id="commit-message-group">
+                        <label for="commit-message-input">Commit Message:</label>
+                        <input type="text" id="commit-message-input" class="full-width">
+                    </div>
+                    <div class="form-group" id="commit-hash-group">
+                        <label for="commit-hash-input">Commit Hash:</label>
+                        <input type="text" id="commit-hash-input" class="full-width">
+                    </div>
+                </div>
+                <div class="form-row">
+                    <div class="form-group">
```

Comment:
Non-blocking comment : I think there is some formatting issue here.

Reply:
Could you please post a screenshot of what you see?

Reply:
Nevermind formatting was messed up in the github not when I pulled the code locally.